### PR TITLE
fix: support text-embedding-ada-002

### PR DESCRIPTION
**Pull request by jgpruitt**

### CODE REVIEW

**Review by cevian — CHANGES_REQUESTED** [review]
Wait how do we know the dimensions of the vector column in the storage table then?

**Comment by jgpruitt** [comment]
Crap. How did this not trip a test?

**Comment by jgpruitt** [comment]
Okay I guess we have to handle this in the vectorizer code

**Review by cevian — APPROVED** [review]

File: projects/extension/sql/ai--0.4.0.sql [hunk 1]
```diff
@@ -2652,14 +2652,6 @@ begin
             raise warning 'one or more grant_to roles do not exist: %', _missing_roles;
```
Comment (cevian) [inline projects/extension/sql/ai--0.4.0.sql:2652]: I think you need to regenerate the sql before merging

**Pull request closed without merging**